website/docs/r/sentinel_log_analystic_workspace_onboard.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
There's a typo in this filename `sentinel_log_analystic_workspace_onboard.html.markdown` -> `sentinel_log_analytics_workspace_onboarding.html.markdown`